apps/aecore/lib/aecore/structures/data_tx.ex

```diff
@@ -3,17 +3,28 @@ defmodule  Aecore.Structures.DataTx do
   Aecore structure of a transaction data.
   """
 
+  alias Aecore.Structures.DataTx
   alias Aecore.Keys.Worker, as: Keys
+  alias Aecore.Chain.ChainState
+  alias Aecore.Structures.SpendTx
   alias Aeutil.Serialization
   alias Aeutil.Parser
 
+  require Logger
+
+  @typedoc "Name of the specified transaction module"
+  @type txType :: SpendTx | atom()
```

Comment:
why do we need the `| atom()`, should be removed or specified as type

Reply:
the naming shouldn't be camelCase, we use underscore_lowercase in general

Reply:
Because we want to say that the txType will not only be SpendTx, but can be something else as well, if we create a new Transaction module

Reply:
what is this something else? should be specified in the type